### PR TITLE
Upgrade to v1.4.9

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem
- *New version upstream*

## Solution
- *Upgrade to v.1.4.9*
- *Add README_fr* 

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/roundcube_ynh%20PR92%20(ericgaspar)/)

### CODE REVIEW

**Review by kay0u — APPROVED** [review]
Finally, the 1.4.9 is out \o/

I propose to merge it as soon as possible since the current version doesn't work.

**Comment by pp-r** [comment]
Sorry for the #94 same as your #92, i'm not very used to Github. I'll look better next time.

**Comment by ericgaspar** [comment]
no worries

File: README.md [hunk 1]
```diff
@@ -3,14 +3,15 @@
 [![Integration level](https://dash.yunohost.org/integration/roundcube.svg)](https://dash.yunohost.org/appci/app/roundcube) ![](https://ci-apps.yunohost.org/ci/badges/roundcube.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/roundcube.maintain.svg)  
 [![Install Roundcube with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=roundcube)
 
+*[Read this readme in english.](./README.md)*
```
Comment (Tomo59) [inline README.md]: shouldn't it be a link to French README ?

**Pull request merged**